service/frontend/service.go

```diff
@@ -88,6 +88,20 @@ func (s *Service) Start() {
 	}
 	metadata = persistence.NewMetadataPersistenceClient(metadata, base.GetMetricsClient())
 
+	metadataV2, err := persistence.NewCassandraMetadataPersistenceV2(p.CassandraConfig.Hosts,
```

Comment:
This is exactly I want to avoid having 2 versions of metadata persistence passed through all layers of the code.

Reply:
maybe we should talk